src/core-services/account/mutations/createGroup.js

```diff
@@ -0,0 +1,72 @@
+import _ from "lodash";
+import Logger from "@reactioncommerce/logger";
+import ReactionError from "@reactioncommerce/reaction-error";
+
+/**
+ * @name group/createGroup
+ * @method
+ * @memberof Group/Methods
+ * @summary Creates a new permission group for a shop
+ * It creates permission group for a given shop with passed in roles
+ * @param {object} input - The input supplied from GraphQL
+ * @param {Object} input.groupData - info about group to create
+ * @param {String} input.groupData.name - name of the group to be created
+ * @param {String} input.groupData.description - Optional description of the group to be created
```

Comment:
If this is optional you should wrap the variable name in brackets `[input.groupData.description]`.

Reply:
done